README.md

```diff
@@ -1,10 +1,40 @@
 # SierraUpdatePoller
```

Comment:
Can you maybe add deployment instructions? Maybe next version if it's too much work.